resources/cron.rb

```diff
@@ -0,0 +1,72 @@
+#
+# Cookbook:: chef-client
+# resource:: chef_client_cron
+#
+# Copyright:: 2020, Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+resource_name :chef_client_cron
```

Comment:
shouldn't this be the resources/chef_client_cron.rb file?

Reply:
oh i see this is against the cookbook, i am not awake.

Reply:
yeah chef-client vs. chef_client